docs/get-started/get-started.md

```diff
@@ -126,10 +126,10 @@ import 'maplibre-gl/dist/maplibre-gl.css';
 </Tabs>
 
 
-## Using with a Compatible Fork
+## Using with a Compatible GL JS Fork
```

Comment:
What does "GL JS" refer to here?
I'd like to avoid changing the heading if possible. It does break existing links in historic issues/discussions.

Reply:
Nothing worth breaking links for - I've reverted the change to the header.